CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Change Log
 
+## [4.0.2](https://github.com/theforeman/puppet-certs/tree/4.0.2) (2018-05-29)
```

Comment:
This might have been lost in a comment, but should we make this a 4.1.0 release?